src/models/ratio.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+
+const ratioSchema = mongoose.Schema({
+    baseCurrencyId: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'Currency'
+    },
+
+    exchangeCurrencyId: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'Currency'
+    },
+
+    ratio: {
+        type: Number,
+        require: true,
+    }
+});
+
+//walletSchema.index({ userId: 1, email: 1 });
+
+const Wallet = mongoose.model("Wallet", walletSchema);
+
+module.exports = Wallet;
```

Comment:
Same here, add a new line at the end of the file
```suggestion
module.exports = Wallet;

```